lib/swagger/swaggerTypes.ts

```diff
@@ -216,9 +216,13 @@ interface BaseSchema {
     modelAsString?: boolean;
     values?: Array<{ value: any; description?: string; name?: string }>;
   };
-  type?: string;
+  type?: string | string[];
```

Comment:
Can you provide an example of multiple types?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/api-scenario/references/v1.2/schema.json#L373